--- t1lib-5.1.2/lib/Makefile.in.orig	2007-12-23 16:49:42.000000000 +0100
+++ t1lib-5.1.2/lib/Makefile.in	2008-01-13 13:29:50.976113311 +0100
@@ -24,7 +24,7 @@
 X_LIBS    = @X_LIBS@
 TOPSRC    = @top_srcdir@
 XPM_LIB   = -lXpm
-XLIB      = @X_PRE_LIBS@ -lXext -lX11 @X_EXTRA_LIBS@
+XLIB      = -lX11 @X_EXTRA_LIBS@
 LDFLAGS   = @LDFLAGS@
 LDLIBS    = @LDLIBS@
 AR        = ar rc
@@ -137,7 +137,7 @@
 	$(LIBTOOL) --mode=link \
 		$(CC) $(LDFLAGS) -o $@ $(T1LIBX_OBJS) \
 	         -version-info @T1LIB_LT_CURRENT@:@T1LIB_LT_REVISION@:@T1LIB_LT_AGE@ \
-	         libt1.la $(X_LIBS) $(XPM_LIB) $(XLIB)  -no-undefined -rpath $(libdir) 
+	         libt1.la $(X_LIBS) $(XLIB)  -no-undefined -rpath $(libdir) 
 	cp t1lib/t1libx.h .
 
 
